sunflower/icons.py

```diff
@@ -117,21 +117,23 @@ def get_mount_icon_name(self, icons):
 
 	def set_window_icon(self, window):
 		"""Set window icon"""
+		base_path = get_static_assets_directory()
+
 		# check system for icon
 		if self.has_icon('sunflower'):
 			window.set_icon(self._icon_theme.load_icon('sunflower', 256, 0))
 
 		# try loading from zip file
 		elif os.path.isfile(sys.path[0]) and sys.path[0] != '':
 			archive = zipfile.ZipFile(sys.path[0])
-			with archive.open('images/sunflower.svg') as raw_file:
+
+			with archive.open(os.path.join(base_path, 'images', 'sunflower.svg')) as raw_file:
```

Comment:
Did you test this change with standalone version? The one generated with `make standalone`? I think it might be breaking it.